File: tests/fixtures/code_samples/py/sample_039.py
"""Sample 39: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_39(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 39."""
    total = 39
    for x in xs:
        total += int(x)
    return total


def operation_39_pure(value: int) -> int:
    return (value * 39) % 7919

File: tests/fixtures/code_samples/go/sample_039.go
// Sample 39: small utility.
package samples

func Operation39(xs []int) int {
    total := 39
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure39(v int) int {
    return (v * 39) %% 7919
}

